### PR TITLE
Add changes for Najdorf branch

### PR DESCRIPTION
**Pull request by Fryguy**

### CODE REVIEW

File: config/labels.yml [hunk 1]
```diff
@@ -122,8 +129,12 @@ release_hammer: &release_hammer
   hammer/backported: "dddddd"
   hammer/conflict: "dddddd"
 
+repo_introduced_in_najdorf: &repo_introduced_in_najdorf
+  <<: *release_najdorf
+
 repo_introduced_in_morphy: &repo_introduced_in_morphy
   <<: *release_morphy
+  <<: *repo_introduced_in_najdorf
```
Comment (chessbyte) [inline config/labels.yml:137]: don't understand how a repo introduced in morphy is also introduced in najdorf

Comment (Fryguy) [reply config/labels.yml:137]: repo_introduced_in_X means you want all the labels for that release and all the ones after it, but not the ones before it.  Put another way a new repo in morphy continues exists in najdorf.  Or, from a label perspective,  a new repo in morphy also acts like a repo that was introduced in najdorf.

The resolved list hopefully helps it make sense:

```ruby
[2] pry(main)> labels = YAML.load_file("config/labels.yml"); nil
=> nil
[3] pry(main)> pp labels["repo_introduced_in_najdorf"]; nil
{"najdorf/yes"=>"000000",
 "najdorf/yes?"=>"000000",
 "najdorf/no"=>"000000",
 "najdorf/backported"=>"000000",
 "najdorf/conflict"=>"000000"}
=> nil
[4] pry(main)> pp labels["repo_introduced_in_morphy"]; nil
{"morphy/yes"=>"000000",
 "morphy/yes?"=>"000000",
 "morphy/no"=>"000000",
 "morphy/backported"=>"000000",
 "morphy/conflict"=>"000000",
 "najdorf/yes"=>"000000",
 "najdorf/yes?"=>"000000",
 "najdorf/no"=>"000000",
 "najdorf/backported"=>"000000",
 "najdorf/conflict"=>"000000"}
=> nil
[5] pry(main)> pp labels["repo_introduced_in_lasker"]; nil
{"lasker/yes"=>"555555",
 "lasker/yes?"=>"555555",
 "lasker/no"=>"555555",
 "lasker/backported"=>"555555",
 "lasker/conflict"=>"555555",
 "morphy/yes"=>"000000",
 "morphy/yes?"=>"000000",
 "morphy/no"=>"000000",
 "morphy/backported"=>"000000",
 "morphy/conflict"=>"000000",
 "najdorf/yes"=>"000000",
 "najdorf/yes?"=>"000000",
 "najdorf/no"=>"000000",
 "najdorf/backported"=>"000000",
 "najdorf/conflict"=>"000000"}
=> nil
```

File: lib/manageiq/release/repo.rb [hunk 2]
```diff
@@ -49,6 +49,7 @@ def fetch(output: true)
 
       def checkout(branch, source = "origin/#{branch}")
         git.reset(:hard => true)
+        git.clean("-xdf")
```
Comment (bdunne) [inline lib/manageiq/release/repo.rb:52]: 🔥 🚽 ❗

Comment (bdunne) [reply lib/manageiq/release/repo.rb:52]: Hope there wasn't anything important there 😄

Comment (Fryguy) [reply lib/manageiq/release/repo.rb:52]: haha...this is actually the thing causing me problems, while simultaneously fixing other problems. 😩

**Comment by Fryguy** [comment]
@chessbyte This is ready to be merged.

**Review by chessbyte — APPROVED** [review]

**Pull request merged**